### PR TITLE
Delete guides error

### PR DESCRIPTION
**Pull request by v-wendysmith**
# Pull request guidance

Thank you for submitting your contribution to our support content! Our team works closely with subject matter experts in CSS and PMs in the product group to review all content requests to ensure technical accuracy and the best customer experience. This process can sometimes take one or more days, so we greatly appreciate your patience.

We also need your help in order to process your request as soon as possible:

- We won't act on your pull request (PR) until you type "**#sign-off**" in a new comment in your pull request (PR) to indicate that your changes are complete.

- After you sign off in your PR, the article will be tech reviewed by the PM or SME if it has more than minor changes. Once the article is approved, it will undergo a final editing pass before being merged.

### CODE REVIEW

**Review by m-hartmann — APPROVED** [review]

**Comment by m-hartmann** [comment]
#sign-off

**Comment by v-wendysmith** [comment]
#sign-off

File: support/dynamics-365/guides/application-functionality/cant-delete-guide.md [hunk 1]
```diff
@@ -1,31 +1,37 @@
...
 ## Resolution
 
-1. In the [Guides model-driven app,](/dynamics365/mixed-reality/guides/model-driven-app-overview) open the guide you're trying to delete.
+> [!NOTE]
+> You need administrator permissions to delete a guide.
+
+To delete a guide, follow these steps:
 
-1. Select the **Related** tab and then **Guide Operator Sessions**.
+1. In the Guides model-driven app, open the guide you try to delete.
```
Comment (v-wendysmith) [inline support/dynamics-365/guides/application-functionality/cant-delete-guide.md:27]: Shouldn't this be "open the guide you tried to delete"? Or you're trying to delete?

**Review by v-wendysmith — COMMENTED** [review]
Hi Lucy. Please see my question. Looks great, thank you!

File: support/dynamics-365/guides/application-functionality/cant-delete-guide.md (hunk 1 above)
Comment (lucciz01) [reply support/dynamics-365/guides/application-functionality/cant-delete-guide.md]: Hi @v-wendysmith, I think your version is better. Here's the suggestion from Bing Chat and I updated back to "you're trying to delete". Thanks for the double-check!

**Comment by lucciz01** [comment]
PR#5210 is created in private repo for this article.

File: support/dynamics-365/guides/application-functionality/cant-delete-guide.md (hunk 1 above)
Comment (v-wendysmith) [reply support/dynamics-365/guides/application-functionality/cant-delete-guide.md]: Thanks Lucy!

**Comment by lucciz01** [comment]
Hi @v-wendysmith, this article is published via private repo. The link will be: https://learn.microsoft.com/en-us/troubleshoot/dynamics-365/guides/application-functionality/cant-delete-guide.

**Pull request closed without merging**